.github/workflows/lint-and-type.yml

```
@@ -7,8 +7,12 @@ on:
     types: [opened, synchronize]
 
 jobs:
-  lint:
+  test:
```

```suggestion
  quality_check:
```

@dai-shi How about this?

---

Hmm, not a big fan.

How about changing `.github/workflows/test-multiple-versions.yml` to have `test_multiple_versions` and `.github/workflows/test-multiple-builds.yml` to have `test_multiple_builds`?

---

@dai-shi Sounds good

---

go ahead.

---

* `.github/workflows/test-multiple-versions.yml`: `version`
* `.github/workflows/test-multiple-builds.yml`: `build`
* `.github/workflows/test-old-typescripts.yml`: `typescript`

@dai-shi How about this?

---

I would like to keep the similarity with yml file names.